src/protagonist/Portal/Pages/Batches/Index.cshtml

```diff
@@ -8,7 +8,7 @@
     var batchId = Model.Batch.Id.GetLastPathElement();
     var percentComplete = Math.Round(Model.Batch.Count > 0 ? (double)Model.Batch.Completed / (double)Model.Batch.Count * 100.0 : 1.0);
     var progressPct = Model.Batch.Finished.HasValue ? 100 : percentComplete;
-    ViewData["Title"] = $"Batch {batchId} - {Model.Batch.Count} images";
+    ViewData["Title"] = $"Batch {batchId} - {Model.Batch.Count} image{(Model.Batch.Count > 1 || Model.Batch.Count == 0 ? "s" : null)}";
```

Comment:
`(Model.Batch.Count > 1 || Model.Batch.Count == 0 ? "s" : null)` is difficult to understand at a glance, is `(Model.Batch.Count == 1 ? null : "s")` clearer?

Reply:
Definitely! Will change this one, thanks for pointing it out.